app/src/main/java/de/westnordost/streetcomplete/quests/building_levels/BuildingLevelsAnswer.kt

```diff
@@ -1,3 +1,3 @@
 package de.westnordost.streetcomplete.quests.building_levels
 
-data class BuildingLevelsAnswer(val levels:Int, val roofLevels:Int?)
+data class BuildingLevelsAnswer(val levels: String, val roofLevels: String?)
```

Comment:
It would be cleaner if the data structure for the answer remained to be an integer.

Reply:
I noticed during testing that it'll crash when a large number (bigger than MAX INT) is entered, because `string.toInt()` throws a `NumberFormatException` then.

Now that we don't convert the string to an integer, just to convert it back to a string when uploading the changes, the issue can't occur anymore.

I agree that `Int` is nicer in the data class though. Should we continue to ignore the crash?

Reply:
Let's ignore it